apps/wallet/src/ui/app/pages/accounts/manage/AccountGroupItem.tsx

```diff
@@ -173,6 +177,11 @@ export function AccountGroupItem({
                                         Export Private Key
                                     </ListItem>
                                 ) : null}
+                                {account.isKeyPairExportable ? (
+                                    <ListItem hideBottomBorder onClick={handleExportPublicKey}>
+                                        Export Public Key
```

Comment:
Don't know if this gets too long, but might be good to have this clarified
```suggestion
                                        Export Public Key With Flag
```

Reply:
<img width="357" height="688" alt="Captura desde 2025-08-05 11-27-38" src="https://github.com/user-attachments/assets/463cec6b-756e-470f-9dd3-99c12cfa4721" />